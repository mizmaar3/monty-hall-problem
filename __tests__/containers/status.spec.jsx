import React from "react";
import { shallow } from "enzyme";
import Status from "containers/status";

describe("Status container", () => {
  it("Should render status with Won bg color", () => {
    const wrapper = shallow(<Status content={"WON"} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("Should render status with lost bg color", () => {
    const wrapper = shallow(<Status content={"LOST"} />);
    expect(wrapper).toMatchSnapshot();
  });
});
