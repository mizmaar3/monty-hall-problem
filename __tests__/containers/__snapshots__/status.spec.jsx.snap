// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status container Should render status with Won bg color 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "rgba(0, 255, 0, 0.2)",
      "display": "flex",
      "fontSize": "50px",
      "height": "100%",
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
    }
  }
>
  WON
</div>
`;

exports[`Status container Should render status with lost bg color 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "rgba(255, 0, 0, 0.2)",
      "display": "flex",
      "fontSize": "50px",
      "height": "100%",
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
    }
  }
>
  LOST
</div>
`;
